runtime/lua/vim/lsp/util.lua

```diff
@@ -384,11 +386,9 @@ function M.text_document_completion_list_to_complete_items(result, prefix)
       dup = 1,
       empty = 1,
       user_data = {
-        nvim = {
```

Comment:
Why do you delete this namespace?

I think the `_resolve_completion_detail` should work with only `nvim-lsp`'s completed item.